org.qelectrotech.QElectroTech.json

```diff
@@ -0,0 +1,131 @@
+{
+  "id": "org.qelectrotech.QElectroTech",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "qelectrotech",
+  "rename-desktop-file": "qelectrotech.desktop",
+  "rename-appdata-file": "qelectrotech.appdata.xml",
+  "rename-icon": "qelectrotech",
+  "copy-icon": true,
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--share=ipc",
+    "--filesystem=host"
+  ],
+  "modules": [
+    {
+      "name": "tkinter",
+      "buildsystem": "simple",
+      "build-commands": [
+         "pip3 install --prefix=${FLATPAK_DEST} ."
+            ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/iwalton3/tkinter-standalone",
+          "commit": "2301112d142ebaf7532b25600c77d1a2edc9ef04"
+        }
+            ],
+      "modules": [
+                {
+          "name": "tcl",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://sourceforge.net/projects/tcl/files/Tcl/8.6.11/tcl8.6.11-src.tar.gz",
+              "sha256": "8c0486668586672c5693d7d95817cb05a18c5ecca2f40e2836b9578064088258"
+            }
+          ],
+          "subdir": "unix",
+          "post-install": [
+              "chmod +w ${FLATPAK_DEST}/lib/libtcl8.6.so"
+          ]
+                },
+                {
+          "name": "tk",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://sourceforge.net/projects/tcl/files/Tcl/8.6.11/tk8.6.11-src.tar.gz",
+              "sha256": "5228a8187a7f70fa0791ef0f975270f068ba9557f57456f51eb02d9d4ea31282"
+            }
+          ],
+          "subdir": "unix",
+          "post-install": [
+              "chmod +w ${FLATPAK_DEST}/lib/libtk8.6.so"
+          ]
+                }
+      ]
+    },
+    {
+      "name": "qelectrotech",
+      "buildsystem": "qmake",
+      "post-install": [
+        "mv ${FLATPAK_DEST}/share/mime/packages/qelectrotech.xml ${FLATPAK_DEST}/share/mime/packages/org.qelectrotech.QElectroTech.xml"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/qelectrotech/qelectrotech-source-mirror.git",
+          "tag": "0.8.0",
+          "commit": "664537c1787d5ca4d5420bb369668d67e9d72d84"
+        },
+        {
+          "type": "patch",
+          "path": "fix_appdata_file.patch"
+        },
+        {
+          "type": "patch",
+          "path": "0001-build-Fix-the-installation-paths.patch"
+        }
+      ]
+    },
+    {
+    "name": "python3.6.0",
```

Comment:
Do you really need this specific version? Is there anything wrong with the one shipped in the runtime? If you need older, why the latest 3.6.x?

Reply:
Requires python 3.5 or above.

Reply:
The runtime ships 3.8.8. Would it be okay?

Reply:
Yes.